### ISSUE TITLE
[EPIC] Source Dremio: built in YAML

### ISSUE THREAD

**Issue by ycherniaiev** [airbytehq/airbyte#18296]
## Tell us about the new connector you’d like to have
**DOCS**
[Xero API Reference][(https://developer.xero.com/documentation/api/accounting/overview](https://docs.dremio.com/cloud/api/))

**Sandbox account creds:** 
LastPass: dremio primary dev account

**Vetting**
- [x] #9031

**MLP**
[Document](https://docs.google.com/document/d/1FpyBVmuTS7XNVBm-dEnYe65ehbfbs05ZZbtSpkYvUmI/edit#)
Partially done (implementation of the Alpha source connector to Alpha needed)

**Blockers** 
- Need help from devs to create a stack on AWS (AWS LastPass: CloudWatch AWS ; Docs https://docs.dremio.com/cloud/signing-up/prerequisites-and-permissions/)

**Comment by arsenlosenko** [comment]
@sherifnada @YowanR here is the [Scoping Document for Dremio](https://docs.google.com/document/d/1Y09BvNXCQEDJK-DakSslXyHaPAxPjwe5Wv9cof6qbCw/), feel free to add any comments or suggestions.

cc @lazebnyi

**Comment by arsenlosenko** [comment]
Here is an up date on situation with sandbox setup for Source Dremio. The goal of the sandbox is to expose their REST API (https://docs.dremio.com/software/rest-api/), endpoints of which is used by connector. There are basically two possible approaches for this:
1. Creating an AWS stack and connecting it to Dremio, as it listed in this doc: https://docs.dremio.com/cloud/signing-up/prerequisites-and-permissions/. It was possible to implement this setup, but it looks like the Cloud instance of Dremio does not have the API endpoints needed, only the Cloud API (https://docs.dremio.com/cloud/api/) is exposed (it could be used to manage project created on their Cloud)
2. Deploying Dremio Docker container on a server. Together with DevOps we managed to deploy this container on our dev infra on AWS (it is a small instance, that does not require a lot of resources), for the sake of testing if this solution works. And it works as expected, the endpoints needed are reachable and they exist. But it was communicated with DevOps that this solution could be only used temporary, because we do not support solutions that require maintenance.

Considering that Solution №2 could only be temporary, I have tried to reach Dremio support about their REST API, to clarify if it exists on Dremio Cloud, because documentation is not clear on that. Right now I have asked @ycherniaiev  to investigate how to create a support request (or reach support somehow), because it is also not clear how to do this from our sandbox account on Dremio Cloud, because it uses a Standard Tier, but support is accessible from Paid Tier (https://www.dremio.com/pricing/).

So, to summarize, right now we have a temporary working solution for sandbox for Dremio, and clarifying with support if Dremio Cloud has the REST API that is needed to be exposed.

cc @lazebnyi @YowanR
